### DISCUSSION TITLE
Issue with IGMP (versions)? - UDM SE v4.0.6

### DISCUSSION THREAD

**Discussion by ARTSRAY** [fabianishere/udm-iptv#347] [Q&A] [answered]
Hi @fabianishere I am struggling to resolve the "Address already in use" error.
IGMP Snooping = enabled
IGMP Proxy I need to check within my UDMSE, (where to find the option?)

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
=== Service Logs ===
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: NATing IPTV network ranges (if necessa            ry)
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: Creating static routes (if necessary)
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: Setting up IGMP Proxy
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: Using improxy...
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: Starting IGMP Proxy
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: MRT_INIT: Address already in use
Aug 20 13:45:24 UDM-SE IMPROXY[225932]: ERROR[main@619]: exiting.........
Aug 20 13:45:24 UDM-SE udm-iptvd[225932]: ERROR[main@619]: exiting.........
Aug 20 13:45:24 UDM-SE systemd[1]: udm-iptv.service: Main process exited, code=e            xited, status=1/FAILURE
Aug 20 13:45:24 UDM-SE systemd[1]: udm-iptv.service: Failed with result 'exit-co            de'.

**Comment by Rinus-86** [comment, marked as answer]
@ARTSRAY I have got aan UDM-PRO but I think what you are look for  is under your WAN settings. It is called IPTV Streaming.

![image](https://github.com/user-attachments/assets/e3ebe74b-5193-43fd-a99b-658a33e458a2)

This is probably on. I had similar issues turning this option of helped.  Don't forget to restart de UDM-ipTV service via the CLI.

**Reply by ARTSRAY** [reply]
That is the exact answer! Issue resolved, thanks a million! Apparently a recent UDM-SE OS firmware update enabled the WAN feature by default.

**Reply by rhedvr** [reply]
Thanks, this was the issue on my Unifi Dream Wall OS 4.0.6 too. Additionally WAN1 is connected to port 20 and had to edit /etc/udm-iptv.conf the WAN1 port 20 = eth19 via Putty.
